homeassistant/components/metoffice/manifest.json

```diff
@@ -3,9 +3,8 @@
   "name": "Met Office",
   "codeowners": ["@MrHarcombe", "@avee87"],
   "config_flow": true,
-  "disabled": "Integration library not compatible with Python 3.12",
   "documentation": "https://www.home-assistant.io/integrations/metoffice",
   "iot_class": "cloud_polling",
   "loggers": ["datapoint"],
-  "requirements": ["datapoint==0.9.8;python_version<'3.12'"]
+  "requirements": ["datapoint==0.9.9"]
```

Comment:
I think you also need to run `python3 -m script.gen_requirements_all` to update `requirements.txt` file. See here: https://developers.home-assistant.io/docs/development_checklist/

Reply:
Took me a couple tries there! I misunderstood if applicable so did it via web. That's done now and generated with the script.